packages/x-data-grid/src/tests/columnHeaders.DataGrid.test.tsx

```diff
@@ -1,14 +1,13 @@
 import * as React from 'react';
-import { createRenderer, fireEvent, screen, within } from '@mui/internal-test-utils';
+import { createRenderer, screen, within } from '@mui/internal-test-utils';
 import { expect } from 'chai';
 import { DataGrid } from '@mui/x-data-grid';
 import { getColumnHeaderCell, getColumnHeadersTextContent } from 'test/utils/helperFn';
-import { fireUserEvent } from 'test/utils/fireUserEvent';
 
 const isJSDOM = /jsdom/.test(window.navigator.userAgent);
 
 describe('<DataGrid /> - Column headers', () => {
-  const { render, clock } = createRenderer({ clock: 'fake' });
+  const { render } = createRenderer();
```

Comment:
Nice, that's even better 👍🏻 I didn't wanna go as far as removing fake clocks in this PR, but eventually we should get rid of them as much as possible.
Thanks for handling this!

Reply:
No problem.
It was a quick change as I saw that there is no reliance on date mocking and/or assertion. 👍